src/presets/deno.ts

```diff
@@ -0,0 +1,35 @@
+import deno from 'rollup-plugin-node-deno'
+import MagicString from 'magic-string'
+import { defineNitroPreset } from '../preset'
+
+export const denoServer = defineNitroPreset({
+  extends: 'node-server',
+  entry: '#internal/nitro/entries/deno-server',
+  commands: {
+    preview: 'deno run --unstable --allow-net --allow-read --allow-env ./server/index.mjs'
```

Comment:
Where can I find `./server/index.mjs` file?

Reply:
When compiled, you can find it in `.output/server/index.mjs`